### PR TITLE
Added Turkish Language Support

### PR DESCRIPTION
**Pull request by cevheri**
Hi,
I added Turkish Language Support and tested.

[issue](https://github.com/fluttercandies/flutter_wechat_assets_picker/issues/330)

Cevheri
Best regards.

### CODE REVIEW

File: lib/src/delegates/asset_picker_text_delegate.dart [hunk 1]
```diff
@@ -844,3 +845,100 @@ class FrenchAssetPickerTextDelegate extends AssetPickerTextDelegate {
   @override
   String get sUnitAssetCountLabel => 'quantité';
 }
+
+/// Added Turkish Language Support
+///
+/// @date 20220601
+/// @author Yusuf Gündoğdu
+/// @author Cevheri
```
Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:853]: We don't use the author field here. Please refer to other delegates to see how to write comments here.

Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:853]: We don't use the author field here. Please refer to other delegates to see how to write comments here.

File: lib/src/delegates/asset_picker_text_delegate.dart [hunk 2]
```diff
@@ -844,3 +845,100 @@ class FrenchAssetPickerTextDelegate extends AssetPickerTextDelegate {
...
+  String get preview => 'Önizleme';
+
+  @override
+  String get select => 'Seç';
+
+  @override
+  String get emptyList => 'Boş liste';
+
+  @override
+  String get unSupportedAssetType => 'Desteklenmeyen HEIC medya tipi!';
```
Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:888]: `!` seems to be unnecessary here, same as other fields with `!`.

Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:888]: `!` seems to be unnecessary here, same as other fields with `!`.

File: lib/src/delegates/asset_picker_text_delegate.dart [hunk 3]
```diff
@@ -844,3 +845,100 @@ class FrenchAssetPickerTextDelegate extends AssetPickerTextDelegate {
...
+
+  @override
+  String get sActionUseCameraHint => 'Kamera Kullan';
+
+  @override
+  String get sNameDurationLabel => 'Süre';
+
+  @override
+  String get sUnitAssetCountLabel => 'Sayı';
+}
```
Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:944]: A proper EOF is required.

Comment (AlexV525) [inline lib/src/delegates/asset_picker_text_delegate.dart:944]: A proper EOF is required.

**Comment by AlexV525** [comment]
Also, could you confirm if Turkish is supported by the TalkBack and the VoiceOver? If it's true, please provide screen records when using them to read those fields. If not, please specify a fallback semantics delegate for the new one.

File: lib/src/delegates/asset_picker_text_delegate.dart (hunk 2 above)
Comment (cevheri) [reply lib/src/delegates/asset_picker_text_delegate.dart]: Completed.

**Comment by cevheri** [comment]
* removed author field in comment
* removed ! in translation word.
* added EOF

**Comment by AlexV525** [comment]
> If it's true, please provide screen records when using them to read those fields.

**Comment by AlexV525** [comment]
Converting to draft until we got more feedback from screen recordings.

**Comment by AlexV525** [comment]
@cevheri Are you still on this?

**Comment by cevheri** [comment]
Yes @AlexV525 
We will test ios

**Comment by cevheri** [comment]
we tested on ios and android, it works.

**Comment by AlexV525** [comment]
Please add a screen record using TalkBack or VoiceOver similar to the one in #424 and resolves conflicts.

**Comment by rei-codes** [comment]
Here it is. I recorded it on my phone
https://github.com/fluttercandies/flutter_wechat_assets_picker/assets/40313719/620b1e28-df97-46f7-8393-24765380b647

**Comment by cevheri** [comment]
fixed conflict

**Review by rei-codes — APPROVED** [review]

**Review by AlexV525 — APPROVED** [review]

**Comment by AlexV525** [comment]
@all-contributors
Add @cevheri as translation.
Add @AlexV525 as a11y, translation.

**Pull request merged**